AGM_Interaction/functions/fn_disarm.sqf

```diff
@@ -0,0 +1,123 @@
+/*
+Author: CAA-Picard
+
+Description:
+Disarm the player
+
+Arguments:
+0: The local unit being disarmed
+1: The unit doing the disarming
+
+Return value:
+None
+*/
+
+private ["_unit", "_guard", "_allGear", "_wh"];
+
+_unit = _this select 0;
+_guard = _this select 1;
+
+_allGear = [
```

Comment:
Reminder: Move [AGM_Respawn_fnc_getAllGear] (https://github.com/KoffeinFlummi/AGM/blob/master/AGM_Respawn/functions/fn_getAllGear.sqf) to Core.